.rubocop.yml

```diff
@@ -4,6 +4,7 @@ require:
   - rubocop-performance
   - rubocop-rails
   - rubocop-minitest
+  - ./lib/custom_cops/action_controller_flash_before_render.rb
```

Comment:
There is already one custom cop in `test/safe_navigation_cop.rb`. Can you please move this new one in there as well? It would probably also fix the complaining codecov problem.

Once battle tested, we can try to port this to upstream.

Reply:
Ah sorry. I didn't see that 😅

Reply:
Fixed!